packages/react-core/src/demos/Animations/examples/Animations.tsx

```diff
@@ -0,0 +1,1699 @@
+              visibility={{
+                default: 'hidden',
+                lg: 'visible'
+              }} /** the settings and help icon buttons are only visible on desktop sizes and replaced by a kebab dropdown for other sizes */
+            >
+              <ToolbarItem>
+                <Button
+                  aria-label="Settings actions"
+                  className="pf-m-settings"
+                  variant={ButtonVariant.plain}
+                  icon={<CogIcon />}
+                />
+              </ToolbarItem>
+              <ToolbarItem>
+                <Button aria-label="Help actions" variant={ButtonVariant.plain} icon={<QuestionCircleIcon />} />
+              </ToolbarItem>
+            </ToolbarGroup>
+          </ToolbarGroup>
+          <ToolbarGroup>
+            <ToolbarItem
+              visibility={{
+                default: 'hidden'
+              }} /** this kebab dropdown replaces the icon buttons and is hidden for desktop sizes */
+            >
+              <Dropdown
+                isOpen={isKebabDropdownOpen}
+                onSelect={onKebabDropdownSelect}
+                onOpenChange={setIsKebabDropdownOpen}
+                popperProps={{ position: 'right' }}
+                toggle={(toggleRef: RefObject<any>) => (
+                  <MenuToggle
+                    ref={toggleRef}
+                    isExpanded={isKebabDropdownOpen}
+                    onClick={onKebabDropdownToggle}
+                    variant="plain"
+                    aria-label="Settings and help"
+                    icon={<EllipsisVIcon />}
+                  />
+                )}
+              >
+                <DropdownList>{kebabDropdownItems}</DropdownList>
+              </Dropdown>
+            </ToolbarItem>
+            <ToolbarItem
+              visibility={{ default: 'hidden', md: 'visible' }} /** this user dropdown is hidden on mobile sizes */
+            >
+              <Dropdown
+                isOpen={isDropdownOpen}
+                onSelect={onDropdownSelect}
+                onOpenChange={setIsDropdownOpen}
+                popperProps={{ position: 'right' }}
+                toggle={(toggleRef: RefObject<any>) => (
+                  <MenuToggle
+                    ref={toggleRef}
+                    isExpanded={isDropdownOpen}
+                    onClick={onDropdownToggle}
+                    icon={<Avatar src={imgAvatar} alt="" size="sm" />}
+                  >
+                    Username
+                  </MenuToggle>
+                )}
+              >
+                <DropdownList>{userDropdownItems}</DropdownList>
+              </Dropdown>
+            </ToolbarItem>
+          </ToolbarGroup>
+        </ToolbarGroup>
+      </ToolbarContent>
+    </Toolbar>
+  );
+
+  const masthead = (
+    <Masthead>
+      <MastheadMain>
+        <MastheadToggle>
+          <PageToggleButton variant="plain" aria-label="Global navigation" isHamburgerButton />
```

Comment:
I'm seeing errors in the console related to the `isHamburgerButton` being passed here without an `isExpanded` prop.